pkg/pipelines/secrets/secrets.go

```diff
@@ -98,6 +98,22 @@ func getClusterPublicKey(service types.NamespacedName) (*rsa.PublicKey, error) {
 	return parseKey(f)
 }
 
+// Retrieves a public key from sealed-secrets-service+1, by finding the
+// service in the provided namespaced name and fetching its key.
+func GetClusterPublicKey(service types.NamespacedName) (*rsa.PublicKey, error) {
```

Comment:
`getClusterPublicKey()` already exists at `pkg/pipelines/secrets/secrets.go`

Reply:
Thanks Chetan, missed that. The changes have been made.